client/views/home.js

```diff
@@ -13,12 +13,41 @@ var TORRENT = fs.readFileSync(
 )
 
 module.exports = function () {
-  var graph = window.graph = new TorrentGraph('.torrent-graph')
+  var graph
+  var hero = document.querySelector('#hero')
+
+  // Don't start the demo automatically on mobile.
+  if (window.innerWidth <= 899) {
+    var beginButton = document.createElement('a')
+    beginButton.href = '#'
+    beginButton.id = 'begin'
+    beginButton.className = 'btn large'
+    beginButton.textContent = 'Begin Demo'
+
+    beginButton.addEventListener('click', function onClick () {
+      beginButton.removeEventListener('click', onClick, false)
+      beginButton.parentNode.removeChild(beginButton)
```

Comment:
Pretty sure that `elem.remove()` is a thing now 👍 https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove But this is fine.